tests/arduino/arduino.ino

```diff
@@ -37,6 +55,8 @@ void loop()
     if (Serial.available() > 0) {
```

Comment:
@vargac I know this is not part of this PR but it might make sense to print some sort of a 'shell character' in front of every command that would it obvious that the shell is waiting for user's input (such as obligatory `$` in (ba)sh, the old version of vic used `+>`, feel free to come up with your own).